### ISSUE TITLE
Readme needs updating because AUR is out of date

### ISSUE THREAD

**Issue by jfabernathy** [Antynea/grub-btrfs#249]
I suggest that the Readme be update for installing on Archlinux because the newest features only get implemented if you manually install from source.  The AUR is flagged out of date.

**Comment by Schievel1** [comment]
I don't understand. I far as I understand it the package in the official Arch repo is grub-btrfs-4.11. https://archlinux.org/packages/community/any/grub-btrfs/
The one on the AUR is a git version (https://aur.archlinux.org/packages/grub-btrfs-git) shouldn't this fetch current HEAD of master branch from here and install it?

In the AUR it's flagged out of date with this message: 
```
 tinys flagged grub-btrfs-git out-of-date on 2022-10-23 (UTC) for the following reason:

    Automatically update grub upon snapshot
```
What is that supposed to mean?

**Comment by jfabernathy** [comment]
All I know is you don't get all the grub-btrfsd.service stuff with  the AUR version. I installed a fresh build of EOS-Xfce4 (Archlinux) and used the source from github. It was right.

**Comment by Schievel1** [comment]
So you’re installing grub-btrfs or grub-btrfs-git?

**Comment by jfabernathy** [comment]
If trying the AUR, I use grub-btrfs. But what works is cloning the repo and make install

**Comment by jfabernathy** [comment]
Maybe I have the name wrong.  The grub-btrfs 4.11-2 you load with pacman  -S is in the community repo and not the AUR. Either way, you need to build from source to get it right

**Comment by jfabernathy** [comment]
BTW, the version when build from source is newer than 4.11-2 that you get from the pacman -S grub-btrfs.  on my system now I'm running:
Version 4.12-master-2022-12-10T21:20:15+00:00

**Comment by Schievel1** [comment]
Jup, but what do you get when installing grub-btrfs-git from the AUR? I think it's the same like installing manually. 

Honestly I don't really know what to do with this. I don't want to change the README because of Arch packaging being slow because I am afraid  to be dragged into changing the README everytime a distro that is listed in the README falls behind in their packaging.

**Comment by jfabernathy** [comment]
Let me setup a VM to test the installation of Archlinux using EndeavourOS and try grub-btrfs-git from AUR instead of grub-btrfs from Community repo.

**Comment by jfabernathy** [comment]
Okay, here's what I found. I installed EOS-xfce4 with btrfs and installed snapper and btrfs-assistant and used btrfs-assistant to setup snapper and take the first snapshot.

Next I install grub-btrfs-git from the AUR.  I warned about out of date and also show 4.11 as the version, but once it was installed it was 4.12.

However, it doesn't install a required dependency inotify-tools and doesn't enable grub-btrfsd.service.

So basically you need to follow the manual install instructions anyway.  Which is what I will be doing until Archlinux fixes the repo.

**Comment by Schievel1** [comment]
This is not an issue of grub-btrfs but an issue of the AUR.
